### DISCUSSION TITLE
Allow Stream Windows to be Natively Snap-able

### DISCUSSION THREAD

**Discussion by dschendt** [f1multiviewer/issue-tracker#364] [Feature requests]
In any other app, I can snap my window to a corner, a half screen, etc. I know the snap options exist in the right-click context menu but they aren't intuitive or work as desired with all aspect ratios. Supporting native window snapping will allow more flexibility and easier layout customization, especially when combined with features like FancyZones in Microsoft's PowerToys utility app.

**Comment by f1multiviewer** [comment]
Hey there, unfortunately, the way Windows forwards the click events (or rather, doesn't) when the whole player is draggable meant that I had to implement a custom window dragging solution, which doesn't work with window snapping. You can still use the right-click menu to use the "Set size" and "Position window" features. Then, saving it in the layout should solve most of your issues.
